models/hivemapper/solana/hivemapper_solana_rewards.sql

```diff
@@ -0,0 +1,49 @@
+{{ 
+    config(
+        schema = 'hivemapper_solana',
+        alias = 'rewards',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'block_slot'],
+        post_hook = '{{ expose_spells(\'["solana"]\',
+                                "project",
+                                "hivemapper",
+                                \'["ilemi", "alexus98"]\') }}')
+}}
+
+with 
+    honey_transfers as (
+        SELECT * FROM {{ ref('tokens_solana_transfers') }} WHERE token_mint_address = '4vMsoUT2BWatFweudnQM1xedRLfJgJ7hswhcpz4xgBTy'
```

Comment:
if we are using incremental filter on tx table below, can we do it here too?
also, can we pull only the columns needed rather than `*`?

Reply:
What if I don't know if other columns from tokens_solana_transfer will be useful for other wizards in the future? For the purpose of mine I could subset, but I guess if anyone dives into hivemapper from a different angle, he might need other columns.

Reply:
okay, I'm on it!